src/components/MultiHopTrade/components/LimitOrder/components/LimitOrderList.tsx

```diff
@@ -155,6 +146,7 @@ export const LimitOrderList: FC<LimitOrderListProps> = ({ cardProps, onBack }) =
                         .div(order.sellAmount)
                         .toNumber()}
                       status={order.status}
+                      onCancel={handleCancelOrder}
```

Comment:
nitpick: do we want to use some case of "cancel initiated" or "cancel intent" terminology of sorts? All this does is setting the order to cancel, not actually cancel it yet

Or perhaps just `onCancelClick` / `handleCancelClick`

Reply:
`handleCancelClick` is nice, lets go with that.

Reply:
2151427d82837a2c91febcaed514dcce0fdc662d